src/blocks/pacman.rs

```diff
@@ -247,13 +218,11 @@ pub async fn run(config: &Config, api: &CommonApi) -> Result<()> {
                     .is_some_and(|regex| has_matching_update(&updates, regex));
                 (values, warning, critical, count)
             }
-            Watched::Both(aur_command) => {
-                let (pacman_updates, aur_updates) = tokio::try_join!(
-                    get_pacman_available_updates(),
-                    get_aur_available_updates(aur_command)
-                )?;
-                let pacman_count = get_update_count(&pacman_updates);
-                let aur_count = get_update_count(&aur_updates);
+            Watched::Both(_) => {
+                let pacman_updates = pacman_backend.get_updates_list().await?;
+                let aur_updates = aur_backend.get_updates_list().await?;
```

Comment:
Please restore the `try_join!` thing to get the updates in parallel. This is also something that we should probably do in the new block too.

Reply:
Done